src/backlight/portfolio/portfolio.py

```diff
@@ -18,21 +18,22 @@ class Portfolio:
     Each element of the list represent the positions of an asset that contribute to the portfolio
     """
 
+    _metadata = ["currency_unit"]
+
     def __init__(self, positions: List[Positions]):
         """
         If there is positions with the same symbol, their value are sum
         """
         symbols = [position.symbol for position in positions]
         assert len(symbols) == len(set(symbols))
+        self.currency_unit = None
```

Comment:
Do we need to add `currency_unit` for Portfolio?

Reply:
Add `currency_unit` in the argument of `__init__`.

Reply:
Also, need to assert `[position.currency_unit for position in positions]` are same.

Reply:
Done!